README.md

```diff
@@ -92,5 +92,7 @@ Ronan Collobert.
 
 ## Versions
 
-The software generally matches the API and implementation of MLX as of tag 
+The ``MLX`` array functions should match MLX as of tag 
+[v0.3.0](https://github.com/ml-explore/mlx/releases/tag/v0.3.0).  The `MLXNN`
+package should match MLX (`mlx.nn`) as of tag
 [v0.0.10](https://github.com/ml-explore/mlx/releases/tag/v0.0.10).
```

Comment:
Update this statement to reflect current state.  MLXNN still needs to be brought up-to-date for any recent changes.